### PR TITLE
Updates StripeSDK to 10.4 + Adds support for SCA

### PR DESCRIPTION
**Pull request by ata-n**

### CODE REVIEW

File: traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 
 public class Router {
-    private static final String BASE_URL = "https://traveler.rc.guestlogix.io/v1";
+    private static final String BASE_URL = "https://traveler.dev.guestlogix.io/v1";
```
[the remark this answers is not in the document]
Comment (ata-n) [reply traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java]: Good catch...I think at the time the updates were only available on dev, but I dont want to commit this. Will confirm and update.

Thanks

File: traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Category.java [hunk 2]
```diff
@@ -20,7 +20,7 @@ public static Category fromString(String value) throws IllegalArgumentException
                 return SHOW;
             case "Event":
                 return EVENT;
-            case "Transfer":
+            case "Transfers":
```
[the remark this answers is not in the document]
Comment (ata-n) [reply traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Category.java]: I think it already was..I remember seeing it

If it wasnt I'll move it to a different commit

File: traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/PaymentError.java [hunk 3]
```diff
@@ -2,16 +2,22 @@
...
 public class PaymentError extends Error {
     public enum Code {
-        PROCESSING_ERROR
+        PROCESSING_ERROR,
+        CONFIRMATION_REQUIRED,
+        CONFIRMATION_FAILED
     }
 
     private Code code;
+    private String data;
```
[the remark this answers is not in the document]
Comment (ata-n) [reply traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/PaymentError.java]: but `null` is an accepted value here

File: traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Traveler.java [hunk 4]
```diff
@@ -364,31 +364,30 @@ protected void main() {
...
      */
     public static void processOrder(Order order, Payment payment, ProcessOrderCallback processOrderCallback) {
-        if (null == localInstance) {
-            processOrderCallback.onOrderProcessError(new TravelerError(TravelerErrorCode.SDK_NOT_INITIALIZED, "SDK not initialized, Initialize by calling Traveler.initialize();"));
-        } else {
-            AuthenticatedUrlRequest request = Router.orderProcess(localInstance.session, order, payment, localInstance.session.getContext());
+        // TODO: Better check and logging for localInstances
+        if (localInstance == null) {
+            return;
+        }
```
[the remark this answers is not in the document]
Comment (ata-n) [reply traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Traveler.java]: I think it is, that PR hasnt been merged yet. I'll wait and rebase should raise this as a conflict

File: traveler_app/src/main/java/com/guestlogix/traveler/network/GuestRoute.java [hunk 5]
```diff
@@ -17,7 +17,7 @@
 
 class GuestRoute {
 
-    private static final String BASE_URL = "https://hklkg7c974.execute-api.ca-central-1.amazonaws.com/dev";
+    private static final String BASE_URL = "https://r3p9qio0x7.execute-api.ca-central-1.amazonaws.com/dev";
```
Comment (Alvtag) [inline traveler_app/src/main/java/com/guestlogix/traveler/network/GuestRoute.java]: I presume this is for DEV?

File: traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Traveler.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (Alvtag) [reply traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Traveler.java]: +1 yep it causes conflict

File: traveler_app/src/main/java/com/guestlogix/traveler/network/GuestRoute.java (hunk 5 above)
Comment (ata-n) [reply traveler_app/src/main/java/com/guestlogix/traveler/network/GuestRoute.java]: Changing back

**Pull request merged**